include/morphio/enums.h

```diff
@@ -60,8 +60,16 @@ enum SectionType {
     SECTION_DENDRITE = 3,         //!< general or basal dendrite (near to soma)
     SECTION_APICAL_DENDRITE = 4,  //!< apical dendrite (far from soma)
 
-    SECTION_GLIA_PERIVASCULAR_PROCESS = 2,
-    SECTION_GLIA_PROCESS = 3,  // TODO: nasty overload there
+    SECTION_GLIA_PERIVASCULAR_PROCESS = 2,  // Note: overlaps with SECTION_AXON
+    SECTION_GLIA_PROCESS = 3,               // Note: overlaps with SECTION_DENDRITE
+
+    SECTION_SPINE_HEAD = 2,  // Note: overlaps with SECTION_AXON
```

Comment:
From last morphology documenttion commit here:
https://bbpcode.epfl.ch/code/#/c/53854/2/source/h5v1.rst

 For dendritic spine the values are: 2: `neck`, 3: `head`.

Reply:
Will fix.